jekyll-watch.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "listen", "~> 3.0"
+  spec.add_runtime_dependency "listen", "~> 3.0", "< 3.1"
```

Comment:
Hm, I'm afraid we'll never remember to remove this constraint. Should we add a comment describing when it can be released, like `# Remove < 3.1 constraint when we drop support for Ruby 2.0`?